### ISSUE TITLE
hexbin function appears to ignore vmin and vmax

### ISSUE THREAD

**Issue by beckermr** [lukelbd/proplot#250]
### Description

The hexbin function appears to ignore vmin and vmax so I cannot see pixels with 1 count.

### Steps to reproduce

A "[Minimal, Complete and Verifiable Example](http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports)" will make it much easier for maintainers to help you.

```python
# your code here
# we should be able to copy-paste this into python and exactly reproduce your bug
```

**Expected behavior**: [What you expected to happen]

**Actual behavior**: [What actually happened]

### Equivalent steps in matplotlib

Please make sure this bug is related to a specific proplot feature. If you're not sure, try to replicate it with the [native matplotlib API](https://matplotlib.org/3.1.1/api/index.html). Matplotlib bugs belong on the [matplotlib github page](https://github.com/matplotlib/matplotlib).

```python
# your code here, if applicable
```

### Proplot version

Paste the result of `import proplot; print(proplot.version)` here.

**Comment by lukelbd** [comment]
Thanks! This was a relatively simple fix (6c9e2eff).

New behavior:

```python
fig, ax = plot.subplots()
m = ax.hexbin(np.random.rand(1000), np.random.rand(1000), gridsize=10, vmin=0, vmax=10)
fig.colorbar(m, loc='bottom')
fig.show()
```

![hexbin](https://user-images.githubusercontent.com/19657652/124362924-62716080-dbf5-11eb-9827-0a0cc02a9f37.png)

**Comment by lukelbd** [comment]
The `no_discrete_norm` option might also be relevant to #233 and #239
